src/Compilers/Core/Portable/Operations/IInterpolatedStringContent.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents a constituent part of an interpolated string.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    public interface IInterpolatedStringContent : IOperation
+    {
```

Comment:
Content is based on the sub-types of IInterpolatedStringContent (string versus interpolated expression)